vignettes/adrs.Rmd

```diff
@@ -95,20 +65,21 @@ read into the environment. This will be a company specific process. Some
 of the data frames needed may be `ADSL`, `RS` and `TU`.
 
 For example purpose, the SDTM and ADaM datasets (based on CDISC Pilot
-test data)---which are included in `{pharmaversesdtm}`---are used.
+test data)---which are included in `{pharmaversesdtm}` and
+`{pharmaverseadam}`---are used.
 
 ```{r message=FALSE}
 library(admiral)
```

Comment:
When we count ANL01FL and there is more than one assessment made on the same day, we take assessment with lowest AVAL.
Now, if for example we have PD and NE on the same day, we would choose NE.
Should we modify aval_resp() function in admiral package?

Reply:
Good point. I have updated the `ANL01FL` derivation. I have not updated the `aval_resp()` function because it is also used for defining `AVAL` for the derived parameters. At Roche we use different functions for `AVAL` and flagging of worst. I have updated the templates and vignettes accordingly.